docs/1.2-DRAFT/data-entities.md

```diff
@@ -174,7 +180,7 @@ If there is no web-accessible description for a file format it SHOULD be describ
 
 ## Core Metadata for Data Entities
 
-The table below outlines the properties that Data Entities, when present, MUST have to be minimally valid.
+The list below outlines the properties that Data Entities, when present in the _RO-Crate Metadata Document_, SHOULD exist, either in `RO-Crate Root` directory or on the web.
```

Comment:
It looks like two different sentences got mixed up in the edit. Also, there is no listing or table below, and not all recommendations are `MUST`. I suggest changing the paragraph to:

Below is an outline of properties that Data Entities, when present in the _RO-Crate Metadata Document_, MUST or SHOULD have. Such data entities should exist, either in the RO-Crate Root directory or on the web.

Reply:
There is no table - but there is a list, I was just trying to reflect that.